_data/locales/de-DE.yml

```diff
@@ -70,6 +70,16 @@ post:
       title: Link kopieren
       succeed: Link erfolgreich kopiert!
 
+# Date time format.
+# See: <http://strftime.net/>, <https://day.js.org/docs/en/display/format>
+df:
+  post:
+    strftime: "%d.%m.%Y"
+    dayjs: "D.M.YYYY"
+  archives:
+    strftime: "%l"
+    dayjs: "M"
+
```

Comment:
```suggestion
```

Please resume these

Reply:
The output of `dayjs` must be the same as `strftime`, otherwise it will jitter the text when loading.

The month of `df.archives` is wrong, you set it to the hour (`%l`), when submitting PR, please test it locally, CI can't check this kind of text content change error.